packages/commons/src/main/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import * as CommonConfig from './config/CommonConfig';
+import * as RandomUtil from './util/RandomUtil';
+import * as StringUtil from './util/StringUtil';
+import * as UrlUtil from './util/UrlUtil';
+
+export = {RandomUtil, StringUtil, UrlUtil, CommonConfig};
```

Comment:
Why do we use this notation (`export =`) and not just `export {...}` for this file?

Reply:
Uh, thx